src/common/FormWrapper.tsx

```diff
@@ -5,7 +5,7 @@ type Props = {
 };
 const FormWrapper = ({ children }: Props) => {
   return (
-    <div className="p-4 mx-auto w-3/4  sm:w-2/4 bg-white mt-2 rounded ">
+    <div className="p-4 mx-auto w-3/4  sm:w-2/4 bg-white mt-2 rounded h-full">
```

Comment:
why we need h-full property here ?

Reply:
@Mubashar1009 its necessary to show form in center vertically